addons/addon-base-post-deployment/packages/base-post-deployment/package.json

```diff
@@ -28,6 +28,7 @@
     "source-map-support": "^0.5.16"
   },
   "scripts": {
+    "test-gh": "NODE_ENV=test jest --config jest.config.js --passWithNoTests --verbose --collectCoverage --watchAll=false --coverage && codecov",
```

Comment:
Should we rename to `coverage` script instead of  `test-gh`  -- doing just coverage?

Reply:
Sure, that makes sense